core-bundle/src/Filesystem/Dbafs/Dbafs.php

```diff
@@ -90,7 +89,7 @@ public function setDatabasePathPrefix(string $prefix): void
 
     public function setMaxFileSize(int $bytes): void
```

Comment:
```suggestion
    /**
     * @deprecated Deprecated since Contao 4.13, to be removed in Contao 5.0.
     */
    public function setMaxFileSize(int $bytes): void
```

So the IDE will also show a deprecation notice.